src/runners/details-feedback.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DetailsFeedbackWidget from '../../components/details_feedback/details_feedback_widget.react';
+
+{
+  const detailsPage = document.querySelector('.details-page');
+  const path = window.location.pathname;
+  const regex = /\/details\/r\/(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9]+)$/;
+
+  if (detailsPage && path.match(regex)) {
+    if (detailsPage.contains(document.getElementById('page_wrap'))) {
+      const pageWrap = document.getElementById('page_wrap');
+      if (!document.getElementById('details-feedback-wrapper')) {
```

Comment:
Hi @gtvj ,

This is to check if the react component mount div is present, and if it not the, it will be created for the react component to mount into.